docs/manual/src/python/configuration.md

```diff
@@ -8,10 +8,34 @@ The generated Python modules can be configured using a `uniffi.toml` configurati
 | ------------------ | -------  |------------ |
 | `cdylib_name`      | `uniffi_{namespace}`[^1] | The name of the compiled Rust library containing the FFI implementation (not needed when using `generate --library`). |
 | `custom_types`      | | A map which controls how custom types are exposed to Python. See the [custom types section of the manual](../udl/custom_types#custom-types-in-the-bindings-code)|
+| `external_packages` | `.` | A map which controls the package name used by external packages.
```

Comment:
This is a bit confusing in the docs, the default is given as `.` but the text says this should be a map.
I would expect that I can set `external_packages = "."` to get the same default behavior, but that doesn't work.
And thus now from only the docs it's not clear how to configure this. It's then explained in the next lines, maybe the table should call this out? "Look further down for details"

Reply:
yeah, thanks, I made a few other tweaks to make that clearer too.